src/ui-components/BalanceInput.tsx

```diff
@@ -134,16 +132,7 @@ const BalanceInput = ({
 			>
 				<Input
 					onBlur={() => onBlur?.()}
-					addonAfter={
-						<div className='flex items-center justify-center gap-1'>
```

Comment:
why we remove this

Reply:
![image](https://github.com/polkassembly/polkassembly/assets/97453078/00d5ce8b-7de1-45f1-916a-9f25d5e3ac76)
icon kha h isme

Reply:
design mai icon ni hai isliye hta diya

Reply:
icon h hato mat